packages/embed/package.json

```diff
@@ -3,13 +3,15 @@
   "version": "1.0.0",
   "private": true,
   "main": "build/index.js",
+  "browser": "dist/webpack/bundle.js",
```

Comment:
I'm curious, how does this work?

Reply:
https://docs.npmjs.com/cli/v6/configuring-npm/package-json#browser

Reply:
This is just a hint field, it doesn't directly do anything but hint that file is supposed to be used in the browser.

Reply:
Oh, okay, I thought it maybe automatically does something :)